engine/source/output/h3d/h3d_build_fortran/create_h3d_quad_scalar.F

```diff
@@ -44,7 +45,7 @@ SUBROUTINE CREATE_H3D_QUAD_SCALAR(H3D_DATA,ID_QUAD_SCALAR,ID_INPUT,TEXT1,STEXT1,
 C-----------------------------------------------
       TYPE (H3D_DATABASE) :: H3D_DATA
       INTEGER ID_QUAD_SCALAR,ID_INPUT,STEXT1,SCOMMENT,LAYER,IR,IS,IT,IUVAR
-      INTEGER IPART(LIPART1,*)
+      INTEGER IPART(LIPART1,*),ID
```

Comment:
INTENT statement is missing

Reply:
Hi @sebastienVilleneuve 

The missing INTENT attributes have been added :-) ! Thanks.

Reply:
Hi @sebastienVilleneuve , 

The squash is done. Sorry for having forgotten this. 

Have a nice afternoon. 

Thanks.